src/fu-util.c

```diff
@@ -2373,6 +2429,18 @@ main (int argc, char *argv[])
 		     /* TRANSLATORS: command description */
 		     _("Activate devices"),
 		     fu_util_activate);
+	fu_util_cmd_array_add (cmd_array,
+		     "get-approved-firmware",
+		     NULL,
+		     /* TRANSLATORS: command description */
```

Comment:
I think that the translators comment might need help here too on what `approved firmware` actually is